content/python/concepts/scope/scope.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'Scope'
+Description: 'The variable is only usable inside the region it is created is called as scope in python'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
```

Comment:
```suggestion
  - `Code Foundations`
  - 'Computer Science'
```

`Data Science` and `Web Development` don't seem close enough in relation to scope in Python. I consider scope to be a foundational coding principle that all programmers, whether they are web developers or data scientists, should understand and appreciate. 😄